### PR TITLE
Pull request adding MFA support

### PR DESCRIPTION
**Pull request by jgrimard**
Hi Alexandr,
https://github.com/balansse/homebridge-vivint/issues/37
Here is a pull request for homebridge-vivint adding MFA support.  
The homebridge-ui used for the login screens is written in TypeScript and React, so it will need to be compiled before publishing to NPM.  To make it easier I created a windows batch script in the /build folder and added some notes in that folder also.
Let me know if you have any questions or any issues.

Thanks,
Jason

### CODE REVIEW

**Comment by arjunmehta** [comment]
@jgrimard instead of the Windows bat script you should just use a regular node script and use `fs` methods for the copies and `child_process` for the executions.

And add a prepublish script to package.json

I can help if you’d like?

File: package.json [hunk 1]
```diff
@@ -1,27 +1,29 @@
 {
   "name": "@balansse/homebridge-vivint",
-  "version": "1.5.4",
+  "version": "1.5.5",
```
Comment (arjunmehta) [inline package.json:3]: Why not a feature release? 1.6.0?

**Comment by jgrimard** [comment]
> @jgrimard instead of the Windows bat script you should just use a regular node script and use `fs` methods for the copies and `child_process` for the executions.
> 
> Then add as a `prepublish` script to package.json
> 
> I can help if you’d like?

Yes please if you want to convert it.  It was more of an afterthought to make publishing to npm easier.

File: package.json (hunk 1 above)
Comment (jgrimard) [reply package.json]: Done

**Comment by arjunmehta** [comment]
@jgrimard Added a PR for this: https://github.com/jgrimard/homebridge-vivint/pull/1/files

**Comment by jgrimard** [comment]
I have merged @arjunmehta's changes into my pull request.  The build scripts are now platform independent and the folder structure is revised.  Tested and all is running smoothly.

**Pull request merged**